cookbook/larger_apps/README.rst

```diff
@@ -1,7 +1,9 @@
 .. _larger_apps:
 
 Building Large Apps
---------------------
+-------------------
+
+.. tags:: Intermediate
```

Comment:
also `Production`?

Reply:
Some of the pages under this section are tagged as "Basic". I think this and sub-pages should be consistent. I'd suggest "Basic" but down to discuss further.

Reply:
My bad. Removed the "Basic" tag.